model/browser/events.yaml

```diff
@@ -0,0 +1,54 @@
+groups:
+  - id: event.browser.exception
+    stability: development
+    type: event
+    name: browser.exception
+    brief: >
+      This event describes an error or exception that occurs in a browser application.
```

Comment:
Lets also "add" a top-level attribute `url.full` (redacted) or include a cutdown `url` within the data), as while having the "filename" is useful, we should also be including the "hosting" page as the source file (bundle) may throw at different locations and for different reasons when used from different pages / scenarios.